src/continuous_agg.h

```diff
@@ -20,10 +20,7 @@
 typedef enum ContinuousAggViewOption
 {
 	ContinuousEnabled = 0,
-	ContinuousViewOptionRefreshLag,
-	ContinuousViewOptionMaxIntervalPerRun,
 	ContinuousViewOptionCreateGroupIndex,
-	ContinuousViewOptionIgnoreInvalidationOlderThan,
 	ContinuousViewOptionMaterializedOnly,
 } ContinuousAggViewOption;
```

Comment:
We don't persist any values referring to these options in any of our catalog tables do we?  It might be safer to just rename these options like `ContinuousViewOptionRefreshLag_deprecated`.

Reply:
@WireBaron The catalog tables still have the values, but they are not used anywhere. I don't understand the idea about renaming to `ContinuousViewOptionRefreshLag_deprecated`. Can you clarify? I think it is better not to provide the options at all, than to allow to set the options without any effect on anything.